lib/signs_ui_web/controllers/messages_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule SignsUiWeb.MessagesController do
+  use SignsUiWeb, :controller
+
+  alias SignsUi.Signs.Messages
+
+  def index(conn, _params) do
+    messages = Messages.list_messages()
+    render(conn, "index.html", messages: messages)
+  end
+
+  def create(conn, params) do
+    case Messages.add_message(params) do
+      {:ok, messages} ->
+        send_resp(conn, 201, "")
+      {:error, _} ->
+        render(conn, "index.html", messages: [])
```

Comment:
I think here we also want just a short `send_resp(conn, 400, "")`.

Reply:
thats a fair point